packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -252,20 +268,22 @@ const _generateStyles = (light: boolean, placeholder: boolean) => {
     } else {
         newStyles = {
             default: {
-                background: white,
-                borderColor: offBlack16,
+                background: invalid ? tokens.color.fadedRed8 : white,
+                borderColor: invalid ? Color.red : offBlack16,
                 borderWidth: 1,
                 color: placeholder ? offBlack64 : offBlack,
             },
             focus: {
-                borderColor: blue,
+                borderColor: invalid ? Color.red : blue,
                 borderWidth: 2,
                 paddingLeft: adjustedPaddingLeft,
                 paddingRight: adjustedPaddingRight,
             },
             active: {
-                background: mix(fade(blue, 0.32), white),
-                borderColor: mix(offBlack32, blue),
+                background: invalid
+                    ? tokens.color.fadedRed
+                    : mix(fade(blue, 0.32), white),
+                borderColor: invalid ? Color.red : mix(offBlack32, blue),
```

Comment:
same suggestion for using `tokens.color` values here instead of `mix`, `fade`, etc. This would be really helpful to start cleaning up the repo from those one-off colors and centralize everything with tokens :)